UE4SS/src/Mod/CppMod.cpp

```diff
@@ -16,20 +16,27 @@ namespace RC
 {
     CppMod::CppMod(UE4SSProgram& program, SystemStringType&& mod_name, SystemStringType&& mod_path) : Mod(program, std::move(mod_name), std::move(mod_path))
     {
-        std::filesystem::path m_dlls_path = m_mod_path;
-
-        m_dlls_path = m_dlls_path / SYSSTR("dlls");
-
-        if (!std::filesystem::exists(m_dlls_path))
+        auto resolved_dlls_path = File::get_path_if_exists(mod_path, SYSSTR("dlls"));
```

Comment:
This is using `mod_path` after it's been moved.
Use `m_mod_path` or `get_name()` instead.
See the `Initialization order` section here for constructor init ordering: https://en.cppreference.com/w/cpp/language/constructor.